assets/js/index.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 require('hof-theme-govuk');
+
+document.getElementById('add-image-button').classList.remove('visuallyhidden');
+
+document.getElementById('add-image-button').addEventListener('click', function imgClick() {
+  document.getElementById('file-upload').click();
+});
+
+document.getElementById('file-upload').addEventListener('change', function fileUploadClick() {
```

Comment:
Sorry to be a pedant, but if you are going to name your functions, use a suitable name. This is bound to trigger on a 'change' event, not a 'click' event

Reply:
No this is totally fair comment.  I'll change